### DISCUSSION TITLE
Update repository URL from GitLab to GitHub

### DISCUSSION THREAD

**Discussion by fiizzy** [codemagic-ci-cd/codemagic-docs#1338] [Q&A] [answered]
Hi, we recently moved our app from Gitlab to GitHub, and I would like to update that URL, which I am unable able to do, because the Update URL form in _App settings > Repository Settings_ is disabled.

How do I go about resolving this issue without having to add a new app and start a fresh Codemagic Configuration on GitHub?
Thanks

**Comment by K-Codemagic** [comment, marked as answer]
Hi can you add it as the new app, and provide me the new and old app ids, so I can make the changes on the backend to keep your old configuration intact.

**Reply by fiizzy** [reply]
Thanks @K-Codemagic. 

Old App
https://codemagic.io/app/632b5fe327eafd4ef10cedf1/workflow/632b5fe327eafd4ef10cedf0/settings

New App
https://codemagic.io/app/63ce821665593f305914247b/workflow/63ce821665593f305914247a/settings

Thanks! Expecting a response.

**Reply by K-Codemagic** [reply]
Hi, I have updated the URL, you can check the old app now. Can you please confirm if the old app is updated with the new repo URL?

**Reply by fiizzy** [reply]
Yes! thank you.
I can confirm that it is updated now!
Do you guys intend on adding this functionality anytime soon? (being able to change it to whatever repository you desire.)

**Reply by K-Codemagic** [reply]
We have that functionality, If the repository is added via GitHub, Bitbucket or GitLab integration, click the Update repository URL button. Codemagic will attempt to update the URL based on the repository ID automatically. However, this is an organizational as well as a migration change. 

I will surely share the feedback with the team

**Reply by fiizzy** [reply]
Oh, thank you! I got it now.
